documentation/modules/overview/con-configuration-points-broker.adoc

```diff
@@ -0,0 +1,33 @@
+// This module is included in:
+//
+// overview/assembly-configuration-points.adoc
+
+// UserStory: Cluster broker configuration
+
+[id="configuration-points-broker_{context}"]
+= Kafka cluster configuration
+
+A Kafka cluster configuration describes how data is stored and accessed.
+
+Storage::
+Kafka and ZooKeeper store data on disks.
++
+{ProductName} requires block storage and is designed to work optimally with cloud-based block storage solutions.
+Three types of data storage are supported:
++
+* Ephemeral (Recommended for development only)
+* Persistent
+* JBOD (Just a Bunch of Disks, suitable for Kafka only)
++
+The persistent storage capacity used by an existing Kafka cluster can be increased.
+
+Listeners:: The following types of listener are supported:
+Listeners configure how clients connect to a Kafka cluster.
++
+* *Plain listener* that does not use encryption
+* *TLS listener* that uses encryption
+* *External listener* for access outside of Kubernetes
+
+Rack awareness:: You can configure a Kafka cluster to run with multiple broker nodes across _racks_.
```

Comment:
H'm. It is one of those kafka-specific configs. I've mentioned racks in the intro to this section. I think it makes sense now, and it might be useful to include it. 

![image](https://user-images.githubusercontent.com/47596553/69447171-d70fc800-0d4d-11ea-83c7-343bffaaece7.png)